test/bundler/bundler_using.test.ts

```
@@ -0,0 +1,227 @@
+  const verdicts = (parse: (code: string) => unknown) =>
+    Object.fromEntries(
+      forms.map(form => {
+        try {
+          parse(`async function f() {\n${form}\n}`);
+          return [form, "accepted"];
+        } catch {
+          return [form, "rejected"];
+        }
+      }),
+    );
+  const jsc = verdicts(code => (0, eval)(`(${code})`));
```

Biome (2.5.7)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)